pysat/_instrument.py

```diff
@@ -941,14 +947,29 @@ def _load_data(self, date=None, fid=None):
 
         if len(fname) > 0:
             load_fname = [os.path.join(self.files.data_path, f) for f in fname]
-            data, mdata = self._load_rtn(load_fname, tag=self.tag,
-                                         sat_id=self.sat_id, **self.kwargs)
-
-            # ensure units and name are named consistently in new Meta
-            # object as specified by user upon Instrument instantiation
-            mdata.accept_default_labels(self)
+            try:
+                data, mdata = self._load_rtn(load_fname, tag=self.tag,
+                                             sat_id=self.sat_id, **self.kwargs)
+                # ensure units and name are named consistently in new Meta
+                # object as specified by user upon Instrument instantiation
+                mdata.accept_default_labels(self)
+                bad_datetime = False
+            except pds.errors.OutOfBoundsDatetime:
+                bad_datetime = True
+                data = self._null_data.copy()
+                mdata = _meta.Meta(units_label=self.units_label,
```

Comment:
Did mdata.accept_devault_labels no longer work?

Reply:
If I just leave mdata.accept_default_labels, I get
```
UnboundLocalError: local variable 'mdata' referenced before assignment
```
Since the code treats finding a corrupted file the same as finding no file, mdata needs to be defined separately.  Open to suggestions for a cleaner implementation.

Reply:
you could probably get away with:
mdata = _meta.Meta().accept_default_labels(self)

Reply:
doesn't work.

Reply:
Ok, leave it as is then.